web-ng/src/app/shared/converters/firebase-data-converter.ts

```diff
@@ -571,11 +582,23 @@ export class FirebaseDataConverter {
     return Role[role].toLowerCase();
   }
 
+  private static isFeatureValid(data: DocumentData): boolean {
```

Comment:
Looks like this isn't used and can be deleted?

Reply:
removed